dttools/src/tlq_config.c

```diff
@@ -15,7 +15,10 @@ char *tlq_config_url( int port, const char *log_path ) {
 	struct link *server;
```

Comment:
Pro-tip -- wherever possible, pass in `stoptime` as a parameter to your function, then just use that `stoptime` for each of the underlying `link` operations.  That way, timeouts get set in one place (in the worker) rather than having constants scattered throughout the code.